terraform/environments/electronic-monitoring-data/cloudwatch-event-rules.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_cloudwatch_event_rule" "definition_update" {
+  name                = "definition-update"
+  schedule_expression = "cron(15 6 * * ? *)" # 06:15 every day
```

Comment:
Why is there a cron job for this? Is this not being run whenever we receive a file? Are we quarantining files for a time before scanning them at this time? I thought we needed some of this data to be available as soon as possible?